reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -249,6 +264,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -FollowRelLink
```

Comment:
Can you add section for `-MaximumFollowRelLink`, default value is Int32.MaxValue

Reply:
This section already exists.